### PR TITLE
Page contato

### PR DESCRIPTION
**Pull request by NazariiSova**

### CODE REVIEW

File: src/components/ContactCard/ContactCard.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <div className="contacts-card-box">
+      <img className="contacts-card-img" src={img} alt={alt}/>
+      <div className="contacts-card-title">{title}</div>
+      <div className="contacts-card-list-box">
+        <ul className="contacts-card-list-location">
+        {listLocation.map((item) => (
+              <li>{item}</li>
+            ))}
+          
+
```
Comment (taras-zherebetskyi) [inline src/components/ContactCard/ContactCard.jsx]: remove blank lines

File: src/components/ContactCard/ContactCard.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+              <li>{item}</li>
+            ))}
+          
+          
+        </ul>
+        <ul className="contacts-card-list-contacts">
+        {listContacts.map((item) => (
+              <li>{item}</li>
+            ))}
+
```
Comment (taras-zherebetskyi) [inline src/components/ContactCard/ContactCard.jsx]: & here

File: src/components/ContactCard/ContactCard.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+export const ContactCard = ({img, alt, title, listLocation, listContacts, time}) => {
+  return (
+    <div className="contacts-card-box">
+      <img className="contacts-card-img" src={img} alt={alt}/>
+      <div className="contacts-card-title">{title}</div>
+      <div className="contacts-card-list-box">
+        <ul className="contacts-card-list-location">
+        {listLocation.map((item) => (
+              <li>{item}</li>
```
Comment (taras-zherebetskyi) [inline src/components/ContactCard/ContactCard.jsx]: react requires the 'key' attribute for the map

File: src/components/ContactCard/ContactCard.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+            ))}
+          
+          
+        </ul>
+        <ul className="contacts-card-list-contacts">
+        {listContacts.map((item) => (
+              <li>{item}</li>
```
Comment (taras-zherebetskyi) [inline src/components/ContactCard/ContactCard.jsx]: react requires the 'key' attribute for the map

File: src/pages/Contato/Contato.page.jsx [hunk 5]
```diff
@@ -1,15 +1,178 @@
...
+
+export const ContatoPage = () => {
+  const [name, setName] = useState("");
+  const [phone, setPhone] = useState("");
+  const [email, setEmail] = useState("");
+  const [message, setMessage] = useState("");
+  const [isFormValid, setIsFormValid] = useState(false);
+
+  function handleNameChange(event) {
+    console.log(event.target.value);
```
Comment (taras-zherebetskyi) [inline src/pages/Contato/Contato.page.jsx]: remove logs for change  handlers

File: src/pages/Contato/Contato.page.jsx [hunk 6]
```diff
@@ -1,15 +1,178 @@
...
+
+  function handleEmailChange(event) {
+    console.log(event.target.value);
+    setEmail(event.target.value);
+  }
+
+  function handleMessageChange(event) {
+    console.log(event.target.value);
+    setMessage(event.target.value);
+  }
```
Comment (taras-zherebetskyi) [inline src/pages/Contato/Contato.page.jsx]: can u create one handler change for all inputs?

File: src/pages/Contato/Contato.page.jsx [hunk 7]
```diff
@@ -1,15 +1,178 @@
...
+            time={"08-18h seg à dom"}
+          />
+          <ContactCard
+            img={sp}
+            alt={"SP"}
+            title={"São Paulo"}
+            listLocation={["Rua Ali Perto, 25", "São Paulo - SP"]}
+            listContacts={["sp@bikcraft.com", "11 99999-9999"]}
+            time={"08-18h seg à dom"}
+          />
```
Comment (taras-zherebetskyi) [inline src/pages/Contato/Contato.page.jsx:183]: can u create array & map it ?😁

**Review by taras-zherebetskyi — CHANGES_REQUESTED** [review]

File: src/components/ContactCard/ContactCard.jsx (hunk 1 above)
Comment (NazariiSova) [reply src/components/ContactCard/ContactCard.jsx]: done

File: src/components/ContactCard/ContactCard.jsx (hunk 2 above)
Comment (NazariiSova) [reply src/components/ContactCard/ContactCard.jsx]: done

File: src/components/ContactCard/ContactCard.jsx (hunk 3 above)
Comment (NazariiSova) [reply src/components/ContactCard/ContactCard.jsx]: done

File: src/components/ContactCard/ContactCard.jsx (hunk 4 above)
Comment (NazariiSova) [reply src/components/ContactCard/ContactCard.jsx]: <ul className="contacts-card-list-contacts" key={title}>
        {listContacts.map((item) => (
              <li>{item}</li>
            ))}

File: src/pages/Contato/Contato.page.jsx (hunk 5 above)
Comment (NazariiSova) [reply src/pages/Contato/Contato.page.jsx]: done

File: src/pages/Contato/Contato.page.jsx (hunk 6 above)
Comment (NazariiSova) [reply src/pages/Contato/Contato.page.jsx]: ok

File: src/pages/Contato/Contato.page.jsx (hunk 7 above)
Comment (NazariiSova) [reply src/pages/Contato/Contato.page.jsx:183]: I don`t want to make this

File: src/pages/Contato/Contato.page.jsx [hunk 8]
```diff
@@ -1,15 +1,189 @@
...
+        </div>
+
+        <form className="contacts-login">
+          <div className="contacts-login-first-inputs">
+            <div className="contacts-login-name-box">
+              <div className="contacts-login-name-title">Nome</div>
+              <input
+                className="contacts-login-name-input"
+                placeholder="Seu nome"
+                onChange={(a) => handlerChangeName(a) }
```
Comment (taras-zherebetskyi) [inline src/pages/Contato/Contato.page.jsx]:
```suggestion
                onChange={ handlerChangeName }
```

**Review by taras-zherebetskyi — CHANGES_REQUESTED** [review]

**Review by taras-zherebetskyi — APPROVED** [review]

**Pull request merged**